mod_reforged/hooks/ui/global/data_helper.nut

```diff
@@ -48,4 +48,13 @@
 		}
 		return ret;
 	}
+
+
+	q.addStatsToUIData = @(__original) function( _entity, _target )
+	{
+		__original(_entity, _target);
+		local properties = _entity.getCurrentProperties();
+		_target.rf_reach <- properties.getReach();
+		_target.rf_reachMax <- 15; // arbitrary
```

Comment:
What do we need `rf_reachMax` for?
I have the feeling a bunch of code in here is from a very early draft of displaying reach and should be removed again.

Reply:
I was fixated on the combat overlay only and couldn't make sense of this change. But it actually belongs to the character screen